src/integ-test/java/com/mathworks/ci/RunMATLABCommandInteg.java

```diff
@@ -0,0 +1,213 @@
+package com.mathworks.ci;
```

Comment:
Thanks for the feedback:) These tests were started off to minimize my manual effort of testing in Jenkins every time I have to do qualification. For now, they dont completely remove the need for interactive validation but I am looking to enhance this test suite further to achieve integration and system testing automation. I am also trying to make these tests helpful for QE's in other teams to run Jenkins build using their parameters. Next thing, I am working to do is invoke local MATLAB instance for these tests.

Having said that, I agree with you that there are similar to Unit Tests. My goal was to capture all system test cases(some test cases might be repetitive considered with unit test cases but not all) and dealing with Test Data combinations. We cannot comment on the probability of catching bugs yet, it might be a useful scenario. Test Data set can always have a lot of permutations to deal with. 

Before we want to merge with our existing plugin, I will enhance the test scenarios and then we could talk about placing them in master.